test/attributes.test.ts

```diff
@@ -141,33 +233,49 @@ describe("Execute AttributeQueries", () => {
     });
 
     test("should execute a RelationshipAttributeQuery", async () => {
-        await createAttribute(client1, {
-            content: {
-                "@type": "RelationshipAttribute",
+        const attributeContent: ConnectorRelationshipAttribute = {
+            owner: client1Address,
+            "@type": "RelationshipAttribute",
+            value: {
+                "@type": "ProprietaryString",
+                title: "text",
+                value: "AGivenName"
+            },
+            key: "key",
+            confidentiality: "public"
+        };
+        await executeFullCreateAndShareRelationshipAttributeFlow(client1, client2, attributeContent);
+
+        const executeRelationshipAttributeQueryResult = await client1.attributes.executeRelationshipAttributeQuery({
+            query: {
+                key: "key",
                 owner: client1Address,
-                value: {
-                    "@type": "ProprietaryString",
-                    title: "ATitle",
-                    value: "AString"
-                },
-                key: "AKey",
-                confidentiality: "public"
+                attributeCreationHints: {
+                    valueType: "ProprietaryString",
+                    title: "text",
+                    confidentiality: "public"
+                }
             }
         });
 
-        const executeRelationshipAttributeQueryResult = await client1.attributes.executeRelationshipAttributeQuery({
+        const executeRelationshipAttributeQueryResult2 = await client2.attributes.executeRelationshipAttributeQuery({
             query: {
-                key: "AKey",
+                key: "key",
                 owner: client1Address,
                 attributeCreationHints: {
                     valueType: "ProprietaryString",
-                    title: "A title",
+                    title: "text",
                     confidentiality: "public"
                 }
             }
         });
+
         expect(executeRelationshipAttributeQueryResult).toBeSuccessful(ValidationSchema.ConnectorAttribute);
 
-        expect(executeRelationshipAttributeQueryResult.result.content.value.value).toBe("AString");
+        expect(executeRelationshipAttributeQueryResult.result.content.value.value).toBe("AGivenName");
+
+        expect(executeRelationshipAttributeQueryResult2).toBeSuccessful(ValidationSchema.ConnectorAttribute);
+
+        expect(executeRelationshipAttributeQueryResult2.result.content.value.value).toBe("AGivenName");
```

Comment:
I don't think we need to duplicate the query here. Querying at one connector completely satisfies this test.